man/org.freedesktop.systemd1.xml

```diff
@@ -3210,6 +3212,8 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2eservice {
 
     <!--property RestrictNamespaces is not documented!-->
 
+    <!--property RestrictFileSystems is not documented!-->
+
```

Comment:
Please add documentation for the dbus API too

I know it's new and everything, but let's try to take the good habits from the start

Reply:
Didn't realize this was new, I thought not documenting it was a conscious decision. Will do.